migration.rst

```diff
@@ -316,19 +312,49 @@ somewhat like this::
 
     class LegacyBridge
     {
-        public static function prepareLegacyScript(Request $request, Response $response, string $publicDirectory): ?string
+
+        /**
+         * Map the incoming request to the right file.  This is the
```

Comment:
I am not sure what the usual convention is, but I think having double space after a colon is not common in the Symfony docs.

```suggestion
         * Map the incoming request to the right file. This is the
```